packages/api-reference/src/components/Content/Content.vue

```diff
@@ -19,6 +23,8 @@ const props = defineProps<{
   layout?: 'default' | 'accordion'
 }>()
 
+useNavigationKeyboardShortcuts()
```

Comment:
This seems like a strange place to initialize. Should this not be done in either the sidebar or ApiReferenceLayout?

Reply:
I moved it around a few times and discussed it with @amritk and we eventually agreed to put it there, because it’s closest to what you’re navigating.

Reply:
I don't mind layout either. Not really a right answer here